prow/pjutil/pjutil.go

```diff
@@ -23,36 +23,52 @@ import (
 	"github.com/satori/go.uuid"
 	"github.com/sirupsen/logrus"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/validation"
 
 	"k8s.io/test-infra/prow/config"
 	"k8s.io/test-infra/prow/github"
 	"k8s.io/test-infra/prow/kube"
 )
 
 const (
-	jobNameLabel = "prow.k8s.io/job"
-	jobTypeLabel = "prow.k8s.io/type"
-	orgLabel     = "prow.k8s.io/refs.org"
-	repoLabel    = "prow.k8s.io/refs.repo"
-	pullLabel    = "prow.k8s.io/refs.pull"
+	defaultProw = "prow.k8s.io"
```

Comment:
Uhhh this is the same as the API Group for a ProwJob -- not something to do with your website or deployment. This should be consistent and I would not consider this something that should be configured

Reply:
The same way that people do not configure the API group for the ProwJob CRD

Reply:
Yeah, this is just a namespaced annotation. I don't think we need this change.

Reply:
reverted